windowsforms/Sudoku/VB/Sudoku.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
Is the solution file needed ?

Reply:
VS creates it automatically when you open a VB project without one. It also holds the readme and .editorConfig file.

Reply:
I think it can be deleted without problems ?
It's preferred to include solution files **only** when it has more than one project.

Reply:
Can I delete it as the last commit before merge. VS Recreates it every time I open the project.

Reply:
Done since there are no more code issues.